### DISCUSSION TITLE
Is there a way to see the current date range that is displayed at the chart?

### DISCUSSION THREAD

**Discussion by zetzetto** [klinecharts/KLineChart#350] [General]
I have a function that has subscribed to **onScroll** action. I want to know what date range is the chart currently displaying.
For example , 
- The chart at initial state was displaying the from 01-11-2022 up to 24-12-2022. 
- After I drag and scroll the chart, now it displays the range of 31-08-2022 up to 24-11-2022. 

Is there a way to find out the current date range displayed at the chart?

**Comment by zetzetto** [comment]
Anyone , any Idea?

**Reply by liihuu** [reply]
Plan to add api `getVisibleRange`.

**Reply by liihuu** [reply]
It has been implemented in the latest version.
